### ISSUE TITLE
solfuzzer crashes in nightly

### ISSUE THREAD

**Issue by Marenz** [ethereum/solidity#13252]
Our nightly test `t_ubu_ubsan_clang_cli` [fails sometimes](https://app.circleci.com/pipelines/github/ethereum/solidity/25120/workflows/e9f4961f-a6aa-4071-8123-22a76515d86b/jobs/1106669)  with 

```
Testing soljson via the fuzzer...
xargs: /root/project/build/test/tools/solfuzzer: terminated by signal 9
```

**Comment by cameel** [comment]
I think it's running out of memory. Same for the codecov job. Using a higher resource class should be enough to fix it.

**Comment by cameel** [comment]
Reopening since this is still happening, just with the other jobs: [`t_ubu_asan_cli`](https://app.circleci.com/pipelines/github/ethereum/solidity/27794/workflows/e61c49a2-1c71-4885-882d-7ec8debb5668/jobs/1233726) and [`t_ubu_codecov`](https://app.circleci.com/pipelines/github/ethereum/solidity/27794/workflows/e61c49a2-1c71-4885-882d-7ec8debb5668/jobs/1233724). They should be adjusted too.

**Comment by cameel** [comment]
Fixed in #13793.
